scripts/parallel-test.sh

```diff
@@ -1,35 +1,20 @@
 #!/bin/bash
-set -e # exit on failure of any "simple" command (excludes &&, ||, or | chains)
-# prepare environment
-PATH=$PATH:~/opt/mongodb/bin
-echo "Extracting build directory..."
```

Comment:
These types of print statements are used for debugging. It is okay if you want to reformat them....    
`echo "[Extracting build directory]"`     
-- or --    
`echo "+++ :compression: Extracting build directory..."`    
...but please do not remove them entirely because when something fails I want people to come up to me saying "My build failed on 'extracting build directory'," rather than "my build crashed before it started". The more information we give engineers, the more they can diagnose themselves.